app/views/candidate_interface/gcse/grade_explanation/_form.html.erb

```diff
@@ -4,11 +4,11 @@
   You need <%= grade_explanation_subject_title(@subject) %> at grade 4 (C) or above, or equivalent
 </h1>
 
-<p class="govuk-body">You can still apply for teacher training if you do not have this. Providers may allow you to sit an equivalency test, or you can retake your GCSE.</p>
-<p class="govuk-body">For advice, speak to a <a href="https://adviser-getintoteaching.education.gov.uk/">teacher training adviser</a>.</p>
+<p class="govuk-body">You can still submit your application. Teacher training providers may allow you to sit an equivalency test, or you can retake your GCSE.</p>
+<p class="govuk-body"><a href="https://getintoteaching.education.gov.uk/#talk-to-us">Chat online</a> for advice.</p>
```

Comment:
Opps, we should have used the helper here, so that the `govuk-link`.

```suggestion
<p class="govuk-body"><%= govuk_link_to('Chat online', t('get_into_teaching.url_online_chat')) %> for advice.</p>
```

Reply:
thanks!